src/Native/Unix/System.Native/pal_networkstatistics.c

```diff
@@ -138,13 +141,18 @@ int32_t SystemNative_GetUdpGlobalStatistics(UdpGlobalStatistics* retStats)
     retStats->IncomingDiscarded = systemStats.udps_noport;
     retStats->IncomingErrors = systemStats.udps_hdrops + systemStats.udps_badsum + systemStats.udps_badlen;
 
+#if defined(__FreeBSD__)
+    // FreeBSD does not have net.inet.udp.pcbcount
```

Comment:
Can we do this based on detection of pcbcount rather than having a platform-specific ifdef?

Reply:
I was thinking about it but t is just string. I can look if we can switch to sysctl() instead of sysctlbyname(). I'll do some more research.